usage/rest-server/src/main/java/org/apache/brooklyn/rest/transform/CatalogTransformer.java

```diff
@@ -63,9 +66,10 @@
         Set<SensorSummary> sensors = Sets.newTreeSet(SummaryComparators.nameComparator());
         Set<EffectorSummary> effectors = Sets.newTreeSet(SummaryComparators.nameComparator());
 
+        EntitySpec<?> spec = null;
+
         try {
-            @SuppressWarnings({ "unchecked", "rawtypes" })
-            EntitySpec<?> spec = (EntitySpec<?>) b.getCatalog().createSpec((CatalogItem) item);
+            spec = (EntitySpec<?>) b.getCatalog().createSpec((CatalogItem) item);
```

Comment:
Causes `Type safety: Unchecked invocation createSpec(CatalogItem) of the generic method createSpec(CatalogItem<T,SpecT>) of type BrooklynCatalog` in Eclipse. That's the reason it was like this before.